compiler/modules/nback.llvm/llvm.bal

```diff
@@ -79,6 +79,7 @@ public type IntrinsicFunctionName "sadd.with.overflow.i64"|"ssub.with.overflow.i
 public class Module {
     private final FunctionDefn[] functions = [];
     private final map<FunctionDecl> functionDecls = {};
```

Comment:
This should be called something like `intrinsicFunctions`.

Reply:
Fixed